test/ssc_test/cmod_utilityrate5_test.cpp

```diff
@@ -430,7 +430,7 @@ TEST(cmod_utilityrate5_eqns, Test_Residential_TOU_Rates_net_billing_carryover_ap
 
     ssc_number_t cost_with_system;
     ssc_data_get_number(data, "elec_cost_with_system_year1", &cost_with_system);
-    EXPECT_NEAR(-173.7, cost_with_system, 0.1);
+    EXPECT_NEAR(-206.61, cost_with_system, 0.1);
 
```

Comment:
Can we add a test folder in SAM with a corresponding rate and values or maybe easier is to use a rate in a SAM project file and then build corresponding ssc tests with that with an Excel workbook as final validation... what do you think?

Reply:
Matching SAM case: https://github.com/NREL/SAM-documentation/blob/master/Unit%20Testing/Utility%20Rates/net_billing_monthly_variation.sam

Excel workbook: https://github.com/NREL/SAM-documentation/blob/master/Unit%20Testing/Utility%20Rates/SAM%202020.11.29%20Rollover%20Month%20Tests/2020.11.29_net_billing_carryover.xlsx